tests/client/core/testErrorHandler.js

```diff
@@ -0,0 +1,140 @@
+import React, { PropTypes } from 'react';
+import { findDOMNode } from 'react-dom';
+import { findRenderedComponentWithType, renderIntoDocument }
+  from 'react-addons-test-utils';
+import { createStore, combineReducers } from 'redux';
+import { Provider } from 'react-redux';
+
+import translate from 'core/i18n/translate';
+import { clearError, setError } from 'core/actions/errors';
+import { ErrorHandler, withErrorHandling } from 'core/errorHandler';
+import errors from 'core/reducers/errors';
+import { createFakeApiError } from 'tests/client/core/reducers/test_errors';
+
+class SomeComponentBase extends React.Component {
+  static propTypes = {
+    errorHandler: PropTypes.object,
+  }
+  render() {
+    return <div />;
+  }
+}
+
+function createErrorStore() {
+  return createStore(combineReducers({ errors }));
+}
+
+function createWrappedComponent({ id, store = createErrorStore() } = {}) {
+  const SomeComponent = translate({ withRef: true })(SomeComponentBase);
+  const ComponentWithErrorHandling =
+    withErrorHandling({ id, name: 'SomeComponent' })(SomeComponent);
+
+  const provider = renderIntoDocument(
+    <Provider store={store}>
+      <ComponentWithErrorHandling />
+    </Provider>
+  );
+  const component = findRenderedComponentWithType(provider, SomeComponent)
+    .getWrappedInstance();
```

Comment:
Oh, huh. All the tests still pass without `.getWrappedInstance()`, nice. Sometimes I get missing property errors without it but I guess I don't fully understand how it works.